impl/client/src/test/javascript/util/xss.spec.js

```diff
@@ -0,0 +1,131 @@
+/*!
+ * Copyright 2024 Hitachi Vantara.  All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+define( "common-ui/util/xss", ( xssUtil ) => {
```

Comment:
Compared to previous (still open) [PR](https://github.com/pentaho/pentaho-platform-plugin-common-ui/pull/1792), the formatting is now not following the set standards. See IntelliJ Check Style in: https://github.com/pentaho/pentaho-coding-standards.